packages/core/addon/services/interfaces/navi-notifications.ts

```diff
@@ -2,11 +2,13 @@
  * Copyright 2021, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
+import { SafeString } from 'handlebars';
+
```

Comment:
Can we use ReturnType until the issue is fixed and there's an import from the `@ember/` namespace

```suggestion
import { htmlSafe } from '@ember/template';

// TODO: Replace once resolved https://github.com/typed-ember/ember-cli-typescript/issues/1155#issue-629409268
export type SafeString = ReturnType<typeof htmlSafe>
```